metalbond/metalbond_client.go

```diff
@@ -171,7 +171,7 @@ func (c *MetalnetClient) removeLocalRoute(destVni mb.VNI, vni mb.VNI, dest mb.De
 		ctx,
 		uint32(vni),
 		&dest.Prefix,
-		dpdkerrors.Ignore(dpdkerrors.NO_VNI, dpdkerrors.ROUTE_NOT_FOUND),
```

Comment:
Did this happen to you? Because this error is not really meant to be ignored, it would indicate a bad call, at least looking at dpservice source code.

Reply:
This change is done as part of migration process, where we are merging relevant osc specific fixes into osc/main branch. @FlorinPeter can give more details on this.